### ISSUE TITLE
Base Image definition - documentation on changes

### ISSUE THREAD

**Issue by tuxianerDE** [lanrat/hass-screenshot#10]
Would it be possible to get a documentation about the changes done? Moving away from a time triggered interval to an event driven is something I am looking for as well.

However I am using the original screenshot tool and not the fork - yet - since I oberserved that at times there are still updates coming to the main version including new versions ob chromium etc. In this case changes made here changes would need to be implemented again. Hence I would like to understand what you actually changed? 

OR would it be possible to define the original screenshot tool as a base image and docker and apply the changes via a shelll script ? - thinking out loudly here

**Comment by lanrat** [comment]
I still actively monitor changes in the upstream version.

Are there specific changes from it you think are needed? most are minor bug fixes that are not applicable with the event driven actions here.

**Comment by tuxianerDE** [comment]
My ask was based on a made expierence. I have started this work with the inkplate on another fork. I then noticed that I had to stick to specific versions of some lovelaces because that fork (and probably chromemium version) did not render these lovelaces properly. 

I moved them to the original (latest) version - your fork is based on too - where then all the stuff started working. That was the reason for my ask, as I wanted to essentially stay in sync and dont bother with "not" updating lovelaces and getting stuck with old versions.
